plugins/inputs/mesos/mesos.go

```diff
@@ -93,7 +105,25 @@ func (m *Mesos) SetDefaults() {
 	}
 }
 
-// Gather() metrics from given list of Mesos Masters
+func prepareAddress(address string) (prepared, host string) {
```

Comment:
this function has an odd signature, I think it should return `(*url.URL, error)`